www/js/control/general-settings.js

```diff
@@ -415,6 +416,15 @@ angular.module('emission.main.control',['emission.services',
         EnketoSurveyLaunch.launch($scope, 'UserProfile');
     };
 
+    $scope.launchEndSurvey = function() {
+        CommHelper.getUser().then(function(profile) {
+            const uuid = profile && profile.user_id && profile.user_id['$uuid'] ? profile.user_id['$uuid'] : 'undefined';
+            const fieldPath = '/aDHunALXzzHXAUMrvkvRT9/_user_id';
+            const returnURL = 'https://reward.amarin.dev';
+            $cordovaInAppBrowser.open(`https://ee.kobotoolbox.org/single/::T9bEosso?d[${fieldPath}]=${uuid}&returnURL=${returnURL}`, '_blank');
```

Comment:
so for the end survey you decided to just use the online kobotoolbox and not embed it into the app?

Reply:
I have no problems with this; just wanted to check

Reply:
Yes, due to the time constraint that we are facing.